XIVComboPlugin/IconReplacer.cs

```diff
@@ -632,6 +632,80 @@ private ulong GetIconDetour(byte self, uint actionID)
                     return SMN.Painflare;
                 }
 
+            // Change Ruin/Outburst into Topaz/Emerald/Ruby Ruin/Outburst
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.SummonerRuinOutburstCombo))
+            {
+                if (actionID == SMN.Ruin1)
```

Comment:
I liked this change so while isn't merged or anything, I checked out the PR, built and tested in lv90 SMN. Didn't test on lower levels yet. 
The outburst part worked, even by dragging tri-disaster (the upgrade) from actions.

Now, SMN ruin goes 1, 2, 3 so testing for ruin1 as you added should've worked I guess? But I'm confused, it didn't for a lv90 SMN which has Ruin3 and at level 90, you can only drag Ruin3 from actions.

I did a quick change on the debug build to test for SMN to `if (actionID == SMN.Ruin3 || actionID == SMN.Ruin1)` and now it replaces Ruin3 correctly /shrug

I have no idea where to get the actionID for Ruin2 in order to add it to `JobActions/SMN.cs` and then add  `actionID == SMN.Ruin2` to the `if` and see if works. 

I still gotta test lower levels but I hope attickdoor does merge. The actions are mutually exclusive, which is the main point of combo.